src/utils/getNumberOfFileChanges.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+import getFullBlame from "./getFullBlame";
+
+async function getNumberOfFileChanges(gitAPI:any, lineNumber:number) {
+    let blamePromises = await getFullBlame(
```

Comment:
You get the blame from here and then blame it again=

Reply:
You're totally right this block can be deleted.